### PR TITLE
feat(webapp): add report a bug

### LINKED ISSUE #296: Add report a Bug / feature request footer link
Link to github to create a new issue

https://github.com/edenia/eden-spend-explorer/issues/new/choose 

Image

### PR DESCRIPTION
**Pull request by JAMares**
### What does this PR do?

- Resolve #296 

### Steps to test

1. Run Project
2. Must see this:
imagen

#### CheckList

- [X] Follow proper Markdown format
- [X] The content is adequate
- [X] The content is available in both english and spanish
- [X] I Ran a spell check

### CODE REVIEW

File: webapp/src/constants/footer.js [hunk 1]
```diff
@@ -36,6 +36,12 @@ export const footerItems = [
         ref: 'https://github.com/edenia/Eden',
         target: '_blank',
         text: 'EdenOS Github Repo'
+      },
+      {
+        underline: 'none',
+        ref: 'https://github.com/edenia/eden-spend-explorer/issues/new/choose',
+        target: '_blank',
+        text: 'Report a bug / Request a Feature'
```
Comment (xavier506) [inline webapp/src/constants/footer.js:44]: Report a **B**ug / Request a Feature

Comment (xavier506) [reply webapp/src/constants/footer.js:44]: please capitalize `Bug`

**Review by xavier506 — CHANGES_REQUESTED** [review]
See comments

File: webapp/src/constants/footer.js [hunk 2]
```diff
@@ -36,6 +36,12 @@ export const footerItems = [
         ref: 'https://github.com/edenia/Eden',
         target: '_blank',
         text: 'EdenOS Github Repo'
+      },
+      {
+        underline: 'none',
+        ref: 'https://github.com/edenia/eden-spend-explorer/issues/new/choose',
+        target: '_blank',
+        text: 'Report a Bug / Request a Feature'
```
Comment (xavier506) [inline webapp/src/constants/footer.js:44]: we should add i18n to the footer

**Review by xavier506 — APPROVED** [review]
LGTM

File: webapp/src/constants/footer.js (hunk 2 above)
Comment (JAMares) [reply webapp/src/constants/footer.js:44]: i go to make a issue for this

**Pull request merged**